doc/api/core/pointsets.rst

```diff
@@ -51,23 +51,38 @@ data:
    demos.plot_datasets('StructuredGrid')
 
 
+The :class:`pyvista.PointSet` (as of ``vtk==9.1.0``) is a concrete class for
+storing a set of points.
+
+.. jupyter-execute::
+   :hide-code:
+
+   import numpy as np
+   import pyvista
+   rng = np.random.default_rng(0)
+   points = rng.random((10, 3))
+   pset = pyvista.PointSet(points)
+   pset.plot()
```

Comment:
One post-final suggestion:
```suggestion
   pset.plot(color='red')
```

The current version looks like this:
![Screenshot from 2022-03-28 23-11-08](https://user-images.githubusercontent.com/17914410/160488245-0c7499aa-6e05-43d0-b917-6fc268e261aa.png)

With red it would look something like this:
![Screenshot from 2022-03-28 23-13-19](https://user-images.githubusercontent.com/17914410/160488242-608a1182-3a99-4d3b-a932-c79342ee7a20.png)

Still small dots, but at least higher contrast. Other colours work too, this was just the first thing that came to mind.